iree/hal/vulkan/registration/driver_module.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2020 Google LLC
```

Comment:
FYI don't bump license years

Reply:
inadvertent (file was created new and then parts were copied in) - not worth commenting on